src/ComposedModal/ComposedModal.svelte

```diff
@@ -75,10 +74,9 @@
   onMount(async () => {
     await tick();
     focus();
-  });
-
-  onDestroy(() => {
-    document.body.classList.remove("bx--body--with-modal-open");
+    return () => {
```

Comment:
The return function will not be triggered if the `onMount` callback is async

Reply:
Okay, does it need to be async?

Reply:
in Modal it's not async either